formatters.Rproj

```diff
@@ -18,3 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace,vignette
```

Comment:
is the vignette actually to be done here? documenting is producing files now whit this. let me remove the vignette. (the following comes from `tern`:
`PackageRoxygenize: rd,collate,namespace`)

Reply:
we can keep this right? there is https://github.com/insightsengineering/formatters/blob/main/vignettes/formatters.Rmd

Reply:
ah. sure. we can remove it then